### ISSUE TITLE
Instant Video Icon

### ISSUE THREAD

**Issue by vaibhavtaylor** [Templarian/MaterialDesign#2777]
Hi,

i wanted an icon for instant video. it is a combination of video and flash icon. can you help me out with this? or you can come up to your best interpretation.

Thanks

**Comment by MrGrigri** [comment]
I am having a difficult time understanding this request. What would this be used for? What is it's function?

**Comment by vaibhavtaylor** [comment]
Hi,

the use is to indicate Instant Video in my app. Instant is normally
indicated by a lightning symbol. i just want that to be inside the Video
Cam Symbol.
Hope this is makes it clear?

Regards,
Vaibhav

**Comment by PeterShaggyNoble** [comment]
I think @MrGrigri was looking for clarification on what you meant by "instant video".

Also, could you provide a use case for it outside the context of your own app?

**Comment by vaibhavtaylor** [comment]
Hi,

the idea here is to indicate Instant Services in any app. like Instant
Chat, Video, call etc used in cases where booking is required before hand.

if this makes it clear. i just want some thing that indicates Instant
Service Idea.

Regards,
Vaibhav

**Comment by MrGrigri** [comment]
The problem is, a lightning symbol is used to indicate a camera/video camera flash.

**Comment by vaibhavtaylor** [comment]
oh yes that is right. can you then help me out on what can be the best way
to show or indicate Instant ?

**Comment by goyney** [comment]
I don't think we have a good answer for this. If you have some ideas, we'd be willing to try them. I do, however, agree that the use of a lightning symbol when paired with a camera indicates flash and would not work well here.
